core/src/main/java/io/fabric8/maven/core/util/validator/ResourceValidator.java

```diff
@@ -101,15 +92,21 @@ private void setupIgnoreRules(ResourceClassifier target) {
      * @throws ConstraintViolationException
      * @throws IOException
      */
-    public int validate() throws ConstraintViolationException, IOException {
-        for (File resource : resources) {
+    public int validate() throws IOException, URISyntaxException {
```

Comment:
Need to fix also `@throws` declarations in comments

Reply:
fixed :+1: